webui/react/src/components/LogViewerLevel.tsx

```diff
@@ -0,0 +1,33 @@
+import { Tooltip } from 'antd';
+import React from 'react';
+
+import { LogLevel } from '../types';
+import { toRem } from '../utils/dom';
+import { capitalize } from '../utils/string';
+
+import Icon from './Icon';
+import css from './LogViewer.module.scss';
+
+export const ICON_WIDTH = 26;
+
+interface Props {
+  logLevel: LogLevel;
+}
+
+const LogViewerLevel: React.FC<Props> = ({ logLevel }) => {
+  const levelStyle = { width: toRem(ICON_WIDTH) };
+
+  const classes = [ css.level ];
+  if (logLevel) classes.push(css[logLevel]);
+
+  return (
+    <Tooltip placement="top" title={`Level: ${capitalize(logLevel || '')}`}>
+      <div className={classes.join(' ')} style={levelStyle}>
+        <div className={css.levelLabel}>&lt;[{logLevel || ''}]&gt;</div>
```

Comment:
**nit:** I know this is code from before, but can `logLevel` ever be `undefined` or `null`? Basically can we just drop the ` || ''` fallback portion?

Reply:
Reading TS interfaces (in particular `TrialLog` from `webui/react/src/types.ts`) it is set to be optional, so that's possible, but on the other side `LogViewerLevel` `props` are setting `logLevel` as required.

So surely this is needing a fix in one of the 2 places.

But looking how API response is converted (`jsonToTrialLog` from `webui/react/src/services/decoder.ts`), `Sdk.V1TrialLogsResponse` is setting `level` as required, but `V1LogLevel` !== `LogLevel`, in fact `decodeV1LogLevelToLogLevel` could potentially return `undefined`.

So I guess we should update TS to set the `undefined` case as expected and keep the fallback? (I did that).

Reply:
But given the other feedback (star instead of the info icon for a default message type) we could simply default to info if undefined?

Reply:
> we could simply default to info if undefined?

I think the info icon as the default works.